syntax = "proto3";
import "google/protobuf/timestamp.proto";

option java_multiple_files= true;
option java_package = "graphlearning.protos";
option java_outer_classname = "EventProtos";

message Event {
  google.protobuf.Timestamp timestamp = 1;
  int32 source = 2;
  int32 target = 3;
  int32 source_label = 4;
  int32 target_label = 5;
  string source_data_hex = 6;
  string target_data_hex = 7;
}
